Simplenote/Classes/SPMarkdownPreviewViewController.m

```diff
@@ -15,13 +15,19 @@
 @import WebKit;
 @import SafariServices;
 
-@interface SPMarkdownPreviewViewController () <WKNavigationDelegate>
-@property (nonatomic, strong) SPVisualEffectView    *navigationBarBackground;
-@property (nonatomic, strong) WKWebView             *webView;
+@interface SPMarkdownPreviewViewController () <WKNavigationDelegate, UIScrollViewDelegate>
+@property (nonatomic, strong) SPBlurEffectView  *navigationBarBackground;
+@property (nonatomic, strong) WKWebView         *webView;
 @end
 
 @implementation SPMarkdownPreviewViewController
 
+- (void)dealloc
+{
+    self.webView.scrollView.delegate = nil;
+    self.webView.navigationDelegate = nil;
+}
+
```

Comment:
These are already weak references. Maybe we don't need to set them to nil in dealloc?

Reply:
You're right!! thank you!